packages/origin-ui-core/src/components/AddDevice.tsx

```diff
@@ -1,5 +1,5 @@
 import React, { useState } from 'react';
-import { IRECDeviceService } from '@energyweb/utils-general';
+import { IRECDeviceService, Countries } from '@energyweb/utils-general';
```

Comment:
We also have a Country Repository https://github.com/energywebfoundation/origin/blob/b6d6609f0031c51e7a6943590b60607e1035ede4/packages/origin-backend/src/pods/country/country.controller.ts

Which one we should use? The static one from utils-general or the one from DB?
@Kuzirashi